code/modules/mob/living/carbon/carbon.dm

```diff
@@ -215,8 +210,8 @@
 			AdjustKnockdown(-3)
 			playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 			M.visible_message( \
-				"<span class='notice'>[M] shakes [src] trying to wake [t_him] up!</span>", \
-				"<span class='notice'>You shake [src] trying to wake [t_him] up!</span>", \
+				"<span class='notice'>[M] shakes [src] trying to wake \him up!</span>", \
```

Comment:
You'd fix your personal history, unless you have multiple forks from your personal repo on this particular branch, and there's a reason not to rewrite it. We can't do that for the main repo. So instead, it becomes
>Commit to change this in a new PR


>this is tested, works, and is part of a larger cleanup of the file.
1. Good.
2. Good.
3. It's not. This PR is for a systems change. You are not cleaning up the file, and you're not making unimportant changes like whitespace or layout.